develop/plone/content/behaviors.rst

```diff
@@ -2,9 +2,9 @@
  Behaviors
 ==============
 
-Behaviors allow you the extend the functionality of existing content.
+Behaviors allow you to extend the functionality of existing content.
 
-* `Tutorial <https://plone.org/products/dexterity/documentation/manual/behaviors>`_
+* :doc:`Tutorial</external/plone.app.dexterity/docs/index>`
```

Comment:
I've always used a space between the link label and the and target reference, but maybe it is not required?

```suggestion
* :doc:`Tutorial </external/plone.app.dexterity/docs/index>`
```